config.yaml

```diff
@@ -132,7 +132,7 @@ params:
   navbar:
     logos:
       github:
-        url: https://github.com/notaryproject/notaryproject
+        url: https://github.com/notaryproject/notation
```

Comment:
I suggest updating this link to `https://github.com/notaryproject/.github`, so that users can read the overview of the Notary Project and intro on each sub-project.

Reply:
+1 on this suggestion

Reply:
agreed.